bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/TabDebugger.java

```diff
@@ -324,7 +331,57 @@ private void createGdbServerGroup(Composite parent) {
 			gd.horizontalSpan = ((GridLayout) comp.getLayout()).numColumns - 1;
 			fGdbServerTclPort.setLayoutData(gd);
 		}
+		{
+			Label label = new Label(comp, SWT.NONE);
+			label.setText(Messages.getString("DebuggerTab.configTargetLabel"));
+			EspConfigParser parser = new EspConfigParser();
+			GridData gd = new GridData();
+			gd.widthHint = 60;
+			gd.horizontalSpan = ((GridLayout) comp.getLayout()).numColumns - 1;
+			fTarget = new Combo(comp, SWT.SINGLE | SWT.BORDER);
+			fTarget.setItems(parser.getTargets().toArray(new String[0]));
```

Comment:
By default, the target should match with the Launch target selected on the toolbar

Reply:
added this functionality with help of the getLaunchTarget method
https://github.com/espressif/idf-eclipse-plugin/blob/878166be511ae39c09d60ded505b9dd07c5b40cb/bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/TabDebugger.java#L577-L590